app/pages/lab-fem/_classifier/tasks/shortcut/editor.jsx

```diff
@@ -0,0 +1,140 @@
+import PropTypes from 'prop-types';
```

Comment:
I'll suggest deleting this file and delete `ShortcutEditor` from app/pages/lab-fem/workflow.cjsx if we know it's deprecated. Seems like a lot to add just to show as deprecated.